benchmark/infrastructure/BenchmarkMeasurementContainer.h

```diff
@@ -343,13 +357,24 @@ class ResultGroup : public BenchmarkMetadataGetter {
                         const std::vector<std::string>& rowNames,
                         const std::vector<std::string>& columnNames);
 
+  /*
+  Delete the given `ResultTable` from the group. Note: Because the group has
+  ownership of all contained entries, this will delete the `ResultTable`.
+  */
```

Comment:
Same comment here.

Reply:
Yes. Changed in [b0dd92b](https://github.com/ad-freiburg/qlever/pull/1207/commits/b0dd92bb0ab3f87f70ce748812b8582a0b432ee4).